File: Javascript/Exercicio07.js
const prompt = require('prompt-sync')();

var valores = new Array(3);
var soma = 0;
for (let i = 0; i < 3; i++) {
    valores[i] = prompt("Informe um valor" + (i + 1) + ": ");
}

valores.forEach(element => {
    element = parseFloat(element);
    soma += element;
});

console.log("Média Aritmética: "+(soma/valores.length));

File: Java/Exercicio07.java
package Java;

import java.util.Scanner;

public class Exercicio07 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double soma= 0;

        double[] valores = new double[3];

        for (int i = 0; i < 3; i++) {
            System.out.print("Informe um valor" + (i + 1) + ": ");
            valores[i] = scanner.nextDouble();
        }
        for (double valor : valores) {
            soma += valor;
        }

        System.out.print("Média Aritmética: "+(soma/valores.length));
        scanner.close();
    }
}
